tests/test_latentdistributiontest.py

```diff
@@ -86,49 +96,51 @@ def test_directed_inputs(self):
         with self.assertRaises(NotImplementedError):
             ldt.fit(A, B)
 
-    def test_SBM_euclidean(self):
-        np.random.seed(12345678)
-        B1 = np.array([[0.5, 0.2], [0.2, 0.5]])
+    def test_SBM_dcorr(self):
+        for test in self.tests.keys():
+            np.random.seed(12345678)
+            B1 = np.array([[0.5, 0.2], [0.2, 0.5]])
 
-        B2 = np.array([[0.7, 0.2], [0.2, 0.7]])
-        b_size = 200
-        A1 = sbm(2 * [b_size], B1)
-        A2 = sbm(2 * [b_size], B1)
-        A3 = sbm(2 * [b_size], B2)
+            B2 = np.array([[0.7, 0.2], [0.2, 0.7]])
+            b_size = 200
+            A1 = sbm(2 * [b_size], B1)
+            A2 = sbm(2 * [b_size], B1)
+            A3 = sbm(2 * [b_size], B2)
 
-        for test in self.tests:
             ldt_null = LatentDistributionTest(
-                test, "euclidean", n_components=2, n_bootstraps=50
+                test, self.tests[test], n_components=2, n_bootstraps=100
             )
             ldt_alt = LatentDistributionTest(
-                test, "euclidean", n_components=2, n_bootstraps=50
+                test, self.tests[test], n_components=2, n_bootstraps=100
             )
             p_null = ldt_null.fit_predict(A1, A2)
             p_alt = ldt_alt.fit_predict(A1, A3)
             self.assertTrue(p_null > 0.05)
             self.assertTrue(p_alt <= 0.05)
 
-    def test_SBM_gaussian(self):
-        np.random.seed(12345678)
-        B1 = np.array([[0.5, 0.2], [0.2, 0.5]])
-
-        B2 = np.array([[0.7, 0.2], [0.2, 0.7]])
-        b_size = 200
-        A1 = sbm(2 * [b_size], B1)
-        A2 = sbm(2 * [b_size], B1)
-        A3 = sbm(2 * [b_size], B2)
-
-        for test in self.tests:
-            ldt_null = LatentDistributionTest(
-                test, "gaussian", n_components=2, n_bootstraps=50
-            )
-            ldt_alt = LatentDistributionTest(
-                test, "gaussian", n_components=2, n_bootstraps=50
-            )
-            p_null = ldt_null.fit_predict(A1, A2)
-            p_alt = ldt_alt.fit_predict(A1, A3)
-            self.assertTrue(p_null > 0.05)
-            self.assertTrue(p_alt <= 0.05)
+    def test_different_sizes(self):
```

Comment:
maybe also want a test for n != m but null is false? just to show power can be greater than 0

Reply:
good point.